Doctests fail on Python 3

Hi,

when disabling the remote tests (#429), I experienced failures in the doctests with Python 3 (3.6.4):
```
=================================== FAILURES ===================================
_________________________ [doctest] asdf.stream.Stream _________________________
010 
011     Used to put a streamed array into the tree.
012 
013     Examples
014     --------
015     Save a double-precision array with 1024 columns, one row at a
016     time::
017 
018          >>> from .asdf import AsdfFile, Stream
UNEXPECTED EXCEPTION: ImportError("cannot import name 'Stream'",)
Traceback (most recent call last):

  File "/usr/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest asdf.stream.Stream[0]>", line 1, in <module>

ImportError: cannot import name 'Stream'

/tmp/asdf-test-bpib0yoa/lib/python3.6/site-packages/asdf/stream.py:18: UnexpectedException
____________________ [doctest] asdf.util.InheritDocstrings _____________________
340 
341     If the class uses multiple inheritance, the docstring will be
342     chosen from the first class in the bases list, in the same way as
343     methods are normally resolved in Python.  If this results in
344     selecting the wrong docstring, the docstring will need to be
345     explicitly included on the method.
346 
347     For example::
348 
349         >>> from .asdf.util import InheritDocstrings
UNEXPECTED EXCEPTION: ModuleNotFoundError("No module named 'asdf.asdf.util'; 'asdf.asdf' is not a package",)
Traceback (most recent call last):

  File "/usr/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest asdf.util.InheritDocstrings[0]>", line 1, in <module>

ModuleNotFoundError: No module named 'asdf.asdf.util'; 'asdf.asdf' is not a package

/tmp/asdf-test-bpib0yoa/lib/python3.6/site-packages/asdf/util.py:349: UnexpectedException
```
With Python 2.7, this runs fine.

Unfortunately, I still run into this problem.
```
=================================== FAILURES ===================================
_________________________ [doctest] asdf.stream.Stream _________________________
010 
011     Used to put a streamed array into the tree.
012 
013     Examples
014     --------
015     Save a double-precision array with 1024 columns, one row at a
016     time::
017 
018          >>> from .asdf import AsdfFile, Stream
UNEXPECTED EXCEPTION: ImportError("cannot import name 'Stream'",)
Traceback (most recent call last):

  File "/usr/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest asdf.stream.Stream[0]>", line 1, in <module>

ImportError: cannot import name 'Stream'

/tmp/asdf-test-djngyzi3/lib/python3.6/site-packages/asdf/stream.py:18: UnexpectedException
____________________ [doctest] asdf.util.InheritDocstrings _____________________
340 
341     If the class uses multiple inheritance, the docstring will be
342     chosen from the first class in the bases list, in the same way as
343     methods are normally resolved in Python.  If this results in
344     selecting the wrong docstring, the docstring will need to be
345     explicitly included on the method.
346 
347     For example::
348 
349         >>> from .asdf.util import InheritDocstrings
UNEXPECTED EXCEPTION: ModuleNotFoundError("No module named 'asdf.asdf.util'; 'asdf.asdf' is not a package",)
Traceback (most recent call last):

  File "/usr/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest asdf.util.InheritDocstrings[0]>", line 1, in <module>

ModuleNotFoundError: No module named 'asdf.asdf.util'; 'asdf.asdf' is not a package

/tmp/asdf-test-djngyzi3/lib/python3.6/site-packages/asdf/util.py:349: UnexpectedException
```

--- a/setup.py
+++ b/setup.py
@@ -130,7 +130,6 @@ setup(name=PACKAGENAME,
       long_description=LONG_DESCRIPTION,
       cmdclass=cmdclassd,
       zip_safe=False,
-      use_2to3=True,
       entry_points=entry_points,
       **package_info
 )

